### ISSUE TITLE
UseReleaseStack option is not documented in FastMM4Options.inc

### ISSUE THREAD

**Issue by jpluimers** [pleriche/FastMM4#45]
A small thing, but please note that `UseReleaseStack` is only mentioned at https://github.com/pleriche/FastMM4/blob/master/FastMM4.pas#L847 but not documented in [FastMM4Options.inc](https://github.com/pleriche/FastMM4/blob/master/FastMM4Options.inc) like `LogLockContention ` is at https://github.com/pleriche/FastMM4/blob/master/FastMM4Options.inc#L310

**Comment by jaclas** [comment]
Yes, please add UseReleaseStack flag to inc file, I found description about it but not configuration for.
